docs/source/guides/getting_started.rst

```diff
@@ -327,9 +329,25 @@ The next chunk of code
 We pull in the properties from the ``build-info.properties`` and make them available to the service
 and more importantly the healthcheck.
 
-In the next binding :java:`bind(HelloWorldResource.class).to(HelloWorldResourceImpl.class);` we are
+AwesomeThingModule
+++++++++++++++++++
+
+.. code-block:: java
+
+  public class AwesomeThingModule extends PrivateModule {
+    protected void configure() {
+      bind(HelloWorldResource.class).to(HelloWorldResourceImpl.class);
+
+      expose(HelloWorldResource.class);
+    }
+  }
+
+``AwesomeThingModule`` is a private module, which contains configuration information for our service. Only the configuration that is explicitly exposed will be available to other modules.
```

Comment:
I think calling the `bindings`/ `injections` as `configurations` can cause some confusion. 

Also, +1 to John's suggestion for adding more information around why private modules.